test/markup_renderer.go

```diff
@@ -174,7 +174,7 @@ func (r *markupRenderer) setScaleModeAttr(attrs map[string]*string, name string,
 }
 
 func (r *markupRenderer) setSizeAttr(attrs map[string]*string, name string, size fyne.Size) {
-	value := fmt.Sprintf("%dx%d", size.Width, size.Height)
+	value := fmt.Sprintf("%dx%d", int(size.Width), int(size.Height))
```

Comment:
That is 1 option. But it led to like 10'000 changes in the text, and the 1.95 vs 2 didn't seem like a helpful change.
If this is a renderer then such things would not be visible. The Gl renderer will similarly round to nearest pixel...

Reply:
Yes, but this is not a renderer but a test helper that shows what a renderer would have as input.
If the renderer accesses some kind of Retina display where it reports a resolution of 1280x800 while having 2560x1600 physical pixels at hand, it might decide to place 19.5 at a different pixel than 19.0.
But even more important: exact numbers give the developer the ability to verify the placement logic with higher accuracy.
I see that the change is huge and therefore I suggest that you create an issue for this change to be done later when #1669 has landed.

Reply:
Good plan

Reply:
Added #1672